src/ConnectedKubernetes/ConnectedKubernetes.Autorest/docs/Set-AzConnectedKubernetes.md

```diff
@@ -0,0 +1,635 @@
+---
+external help file:
+Module Name: Az.ConnectedKubernetes
+online version: https://learn.microsoft.com/powershell/module/az.connectedkubernetes/set-azconnectedkubernetes
+schema: 2.0.0
+---
+
+# Set-AzConnectedKubernetes
+
+## SYNOPSIS
+API to set properties of the connected cluster resource
+
+## SYNTAX
+
+### SetExpanded (Default)
+```
+Set-AzConnectedKubernetes -ClusterName <String> -ResourceGroupName <String> -Location <String>
```

Comment:
Please use singular name

Reply:
@msJinLei  We kept the plural form in the name to maintain consistency with the other commands in this module.:
```
Get-AzConnectedKubernetes
New-AzConnectedKubernetes
Remove-AzConnectedKubernetes
Update-AzConnectedKubernetes
```

We've also put an exception to suppress the analysis check for plural in [this commit](https://github.com/Azure/azure-powershell/pull/26026/commits/37956893a05c0b4801be367fc20e91cb23a0b54e)